README.md

```diff
@@ -23,17 +23,17 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.nexmo:nexmo-spring-boot-starter:1.1.0'
+    implementation 'com.vonage:nexmo-spring-boot-starter:2.2.0'
```

Comment:
this version should be 2.0.0 - no need to jump a couple minor versions ahead as this is semver.

Reply:
2.0.0 is correct